bay/plugins/mounts.py

```diff
@@ -25,26 +25,43 @@ def load(self):
 
 
 @click.command()
+@click.option("--profile-only", "-p", default=False, is_flag=True)
 @click.pass_obj
-def mounts(app):
+def mounts(app, profile_only):
     """
     List all current dev mounts.
     """
     dev_mounts = defaultdict(dict)
+
     for container in app.containers:
         unmounted_devmodes = set(container.devmodes.keys())
         runtime_options = app.containers.options(container)
         if runtime_options:
             devmodes = app.containers.options(container).get('devmodes')
             dev_mounts[container.name]['mounted'] = sorted(devmodes)
             dev_mounts[container.name]['unmounted'] = unmounted_devmodes.difference(devmodes)
+        else:
+            if profile_only:
```

Comment:
why not do `if not profile_only:` so we avoid continue here and we just leave everything else as is ?